app/controllers/file.js

```diff
@@ -17,6 +17,39 @@ const FileController = function(url, header) {
 
 util.inherits(FileController, EventEmitter);
 
+/* Download file and save to disk. */
+FileController.prototype.downloadFile = function() {
```

Comment:
@donnapep @stulees I agree, we should just have a function for downloading with request and use the writeFile function to write to the file. It also should not change the proxy. 

@donnapep can you leave the code as it is with the proxy then just add a new function called downloadFile. This function calls writeFile when a 200 happens. Then, this function is called when checking if there no file on cache folder and download folder. 

cheer